### PR TITLE
Remove references to sata132/jwalker file paths.

### PR DESCRIPTION
**Pull request by jasonwalker80**
The repeat masker file was moved.  Instead of a default_value it's switched to an example_value.

For the capture tool, the default_value was simply removed.

### CODE REVIEW

**Comment by jasonwalker80** [comment]
CI, test this.

File: lib/perl/Genome/Model/Tools/BioSamtools/RepeatContent.pm [hunk 1]
```diff
@@ -15,8 +14,7 @@ class Genome::Model::Tools::BioSamtools::RepeatContent {
         repeats_file => {
             doc => 'A RepeatMasker table of repeats for the genome.',
             is => 'Text',
-            default_value => $DEFAULT_REPEATS_FILE,
-            is_optional => 1,
+            example_value => '/gscmnt/gc13001/info/model_data/jwalker_scratch/RepeatEval/hg19.fa.out',
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/BioSamtools/RepeatContent.pm]: I believe this must be `example_values`.

**Comment by jasonwalker80** [comment]
Thanks @tmooney!  I wasn't sure of the error at first and I was mistakenly running tests from my master branch.  The example_values syntax has been updated and tests are now passing.

**Comment by tmooney** [comment]
+1 🐫

**Comment by acoffman** [comment]
+1

**Pull request merged**